cartography/intel/azure/key_vaults.py

```diff
@@ -279,27 +350,28 @@ def sync_key_vaults(
             has_next_page = True
             key_vaults_list = key_vaults_list[page_start:page_end]
             common_job_parameters['pagination']['key_vaults']['hasNextPage'] = has_next_page
-
+    load_key_vaults(neo4j_session, subscription_id, key_vaults_list, update_tag)
     for key_vault in key_vaults_list:
         # KEY VAULT KEYS
-        key_client = get_key_vault_keys_client(credentials, key_vault.get('properties',{}).get('vault_uri',None))
-        keys = get_key_vault_keys_list(key_client, key_vault.get('id',None), regions, common_job_parameters)
+        keys = get_key_vault_keys_list(client, key_vault)
```

Comment:
I thought we need to call `update_access_policies` first before we access keys, secrets, certificates, etc. will this return any data if `update_access_policies` is not called for the vault yet?

Reply:
to get list of keys and secrets we don’t need of `update_access_policies`,this needs only for to get list of certificates